openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -29,16 +29,15 @@ RUN mkdir -p /openhands && \
 # Directory containing subdirectories for virtual environment.
 ENV POETRY_VIRTUALENVS_PATH=/openhands/poetry
 
-RUN if [ ! -d /openhands/miniforge3 ]; then \
-    wget --progress=bar:force -O Miniforge3.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh" && \
-    bash Miniforge3.sh -b -p /openhands/miniforge3 && \
-    rm Miniforge3.sh && \
-    chmod -R g+w /openhands/miniforge3 && \
-    bash -c ". /openhands/miniforge3/etc/profile.d/conda.sh && conda config --set changeps1 False && conda config --append channels conda-forge"; \
-    fi
+# Install micromamba
+RUN mkdir -p /openhands/micromamba/bin && \
+    /bin/bash -c "PREFIX_LOCATION=/openhands/micromamba BIN_FOLDER=/openhands/micromamba/bin INIT_YES=no CONDA_FORGE_YES=no $(curl -L https://micro.mamba.pm/install.sh)"
```

Comment:
`CONDA_FORGE_YES` sets the channel to `conda-forge`, can this be a "yes" rather than a "no"? We do use this channel below.

Reply:
good catch 👍 (what's wrong with me today lol)